src/components/ActionBar/style.css

```diff
@@ -0,0 +1,15 @@
+.base {
```

Comment:
hopefully none to be honest. The goal was to have no impact on the final style, only on the selectors. The actual standard of the prefixing is probably the point of discussion (which is on the [webpack config](https://github.com/onfido/onfido-sdk-ui/pull/52/files#diff-3fe40ded64ba5bb89dcae9266f8dc418R26)).

Another point to consider is also the way that common style is imported and how it can be overriden. At the moment the order of the import plays a big role and maybe it shouldn't. [See here](https://github.com/onfido/onfido-sdk-ui/pull/52/files#diff-ed0f78e51202e84128626370456b9695R9), `style.icon` overrides `theme.icon` only because of the order of importing above.